recipes/c-ares/all/conanfile.py

```diff
@@ -61,11 +62,14 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
         tools.rmdir(os.path.join(self.package_folder, "share"))
+        for pdb_file in glob.glob(os.path.join(self.package_folder, "bin", "*.pdb")):
+            os.remove(pdb_file)
```

Comment:
```suggestion
        tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), "*.pdb")
```

Reply:
Please set Conan minimum version to `1.29.1` when using this

Reply:
another questions for #3614